README.md

```diff
@@ -202,97 +202,102 @@ Note: this gas strategy is used by keeper in all interactions with chain but whe
+keepers.  **Kicks are not sharded**; for an auction contract, only one keeper should be configured to `kick`.
+
+If you are sharding across multiple accounts, you may wish to have another account handle all your `deal`s.  The
+`--deal-for` argument allows you to specify a space-delimited list of accounts for which you'll deal auctions.  You
+may disable dealing auctions by specifying `--deal-for NONE` in each of your shards.  If you'd like to donate your gas
+to deal auctions for all participants, `--deal-for ALL` is also supported.  Unlike kicks, **deals are sharded**, so
+remove sharding configuration if running a dedicated deal keeper.
+
+Too many pending transactions can fill up the transaction queue, causing a subsequent transaction to be dropped.  By
+waiting a small `--bid-delay` after each bid, multiple transactions can be submitted asynchronously while still
+allowing some time for older transactions to complete, freeing up the queue.  Many parameters determine the appropriate
+amount of time to wait.  For illustration purposes, assume the queue can hold 12 transactions, and gas prices are
+reasonable.  In this environment, a bid delay of 1.2 seconds might provide ample time for transactions at the front of
+the queue to complete.  [Etherscan.io](etherscan.io) can be used to view your account's pending transaction queue.
+
 #### Hardware and operating system resources
 
  * The most expensive keepers are `flip` and `flop` keepers configured to `kick` new auctions.
  * To prevent process churn, ensure your pricing model stays running for a reasonable amount of time.
- 
- 
+
+
 ## Infrastructure
 
-This keeper connects to the Ethereum network using [Web3.py](https://github.com/ethereum/web3.py) and interacts with 
-the Dai Stablecoin System (DSS) using [pymaker](https://github.com/makerdao/pymaker).  A connection to an Ethereum node 
-(`--rpc-host`) is required.  [Parity](https://www.parity.io/ethereum/) and [Geth](https://geth.ethereum.org/) nodes are 
-supported over HTTP.  Websocket endpoints are not supported by `pymaker`.
+This keeper connects to the Ethereum network using [Web3.py](https://github.com/ethereum/web3.py) and interacts with
+the Dai Stablecoin System (DSS) using [pymaker](https://github.com/makerdao/pymaker).  A connection to an Ethereum node
+(`--rpc-host`) is required.  [Parity](https://www.parity.io/ethereum/) and [Geth](https://geth.ethereum.org/) nodes are
+supported over HTTP. Websocket endpoints are not supported by `pymaker`.
+
+If you don't wish to run your own Ethereum node, third-party providers are available.  This software has been tested
+with [ChainSafe](https://chainsafe.io/) and [QuikNode](https://v2.quiknode.io/). Infura is incompatible, however, because it does not support the `eth_sendTransaction` RPC method, which is [utilized in](https://github.com/makerdao/pymaker/blob/69c7b6d869bb3bc9c4cca7b82cc6e8d435966d4b/pymaker/__init__.py#L431) pymaker.
```

Comment:
Recommend wrapping lines to 120 chars for consistency.  I know this isn't followed a few other places in the file.